docs/content/v0.20/packaging.md

```diff
@@ -211,11 +216,24 @@ This chart creates (among other things) a `Deployment` object that runs a contai
 The chart is published at the OCI repository
 `ghcr.io/kubestellar/kubestellar/kubestellar-operator-chart`.  What stores the chart there?
 
+### clusteradm container image
+
+The kubestellar GitHub repository has a script,
+`hack/build-clusteradm-image.sh`, that creates and publishes a
+container image holding the `clusteradm` command from OCM. The source
+of the container image is read from the latest release of
+[github.com/open-cluster-management-io/clusteradm](https://github.com/open-cluster-management-io/clusteradm),
+unless a command line flag says to use a specific version. This script
+also pushes the built container image to
+[quay.io/kubestellar/clusteradm](https://quay.io/repository/kubestellar/clusteradm)
+using a tag that equals the ocm/clusteradm version that the image was
+built from.
+
 ### KubeFlex PostCreateHooks
 
 There are two `PostCreateHook` objects defined in [config/postcreate-hooks](../../../config/postcreate-hooks). What uses these copies of those hook definitions?
 
-- `ocm.yaml` adds `clusteradm` by running a container using the image `quay.io/kubestellar/clusteradm:0.7.2` which is built from [the OCM source](https://github.com/open-cluster-management-io/clusteradm) using the script [build-clusteradm-image.sh](../../../hack/build-clusteradm-image.sh).
+- `ocm.yaml` adds `clusteradm` by running a container using the image `quay.io/kubestellar/clusteradm:0.7.2`; see [above](#clusteradm-container-image) about the source of that.
 - `kubestellar.yaml` runs container image `quay.io/kubestellar/helm:v3.14.0` (which is built from [the Helm source](https://github.com/helm/helm/tree/v3.14.0) by a process that we need to document) to instantiate the chart from `oci://ghcr.io/kubestellar/kubestellar/kubestellar-operator-chart` with chart version `0.20.0-alpha.1`, which is built by (what?) from (what? probably answered above).
```

Comment:
As for the building helm image, isn't that done in the script https://github.com/kubestellar/kubestellar/blob/main/hack/build-helm-image.sh ?

Reply:
Thanks. I will update that in a future PR.